skyportal/handlers/api/user.py

```diff
@@ -1,40 +1,82 @@
 from ..base import BaseHandler
 from baselayer.app.access import permissions
 from ...models import DBSession, User, Group, GroupUser, cfg
-from ...model_util import role_acls
 
-from sqlalchemy.orm import joinedload
+
+def add_user_and_setup_groups(username, roles, group_ids_and_admin):
```

Comment:
We need to allow users to specify a list of `[group_id, admin]` (`[int, bool]`) pairs in the user API so that one can add a user and add them to the specified groups (some of which they might be admins) in a single request. What would be a better approach here? A list of objects? e.g. `[{"group_id": 1, "admin": false}, ...]`

Reply:
Ah, I see!  No problem, maybe just clarify that in the docstring.